src/main/java/namor/template/stack/ArrayStack.java

```diff
@@ -0,0 +1,72 @@
+package namor.template.stack;
+
+import java.util.Arrays;
+
+public class ArrayStack {
+
+    private int top;
+    private int size;
+    private int[] stack;
+    private static final int SIZE = 10;
+
+    public ArrayStack(int size) {
+        if (size < 1) {
+            throw new IllegalArgumentException("Can't create stack with size less than 1");
```

Comment:
Not a best message. For now left it as it is but fo future pay attantion on logs when you get errors how better to write meaningful message. The core problem here that you do not paste wrong value in message. Better to message something like "Some value x is not correct. Should be between this and that", and this is not the only way to write messages, the core concepr here is that the message better to write as short as it's possible and maximum informative.